lib/customer.rb

```diff
@@ -0,0 +1,43 @@
+require 'csv'
+require 'awesome_print'
+
+module Grocery
+  class Customer
+    attr_reader :id, :email, :address
+
+    def initialize(customer_id, email, address)
+      @id = customer_id
+      @email = email
+      @address = address
+    end
+
+    def self.all
+      all_customers = []
+      CSV.read("/Users/brandyaustin/ada/week2/grocery_store/grocery-store/support/customers.csv").each do |row|
```

Comment:
This type of "absolute" path in your program you should avoid at all costs.  It only works on **your** machine.

You should instead make the path "relative" to where the user is running the app (the main project folder).

It should be

```ruby 
CSV.read("support/customers.csv").each do |row|
```